src/styles/blocks/card.scss

```diff
@@ -0,0 +1,105 @@
+.card {
+  display: flex;
+  flex-direction: column;
+  box-sizing: border-box;
+  width: 200px;
+  border: 1px solid $card-border-color;
+  border-radius: 5px;
+  padding: 32px 16px 16px;
+  transition: 300ms;
```

Comment:
```suggestion
 transition: scale 0.3;
```

Reply:
Misteriously, this improvement doesnt't work, totally cancelling the duration of scale.